include/envoy/ssl/context_config.h

```diff
@@ -111,6 +111,16 @@ class ClientContextConfig : public virtual ContextConfig {
    *         for names.
    */
   virtual const std::string& signingAlgorithmsForTest() const PURE;
+
+  /**
+   * Check whether TLS Certificate entity was being extracted from SDS server.
+   */
+  virtual bool checkTlsCertificateEntityExists() const PURE;
+
+  /**
+   * Check whether certificate validation context entity was being extracted from SDS server.
+   */
+  virtual bool checkCertificateValidationContextEntityExists() const PURE;
```

Comment:
do we need these to be separate? see comment below.

Reply:
done